### PR TITLE
`cellfinder` v1.3.0 blog post

### LINKED ISSUE #177: cellfinder v1.3.0 blog post
The next version of cellfinder is likely to be a (relatively) large one, and we should write a blog post to outline the differences. This will include:
- Switch to torch backend (https://github.com/brainglobe/cellfinder/pull/396)
- Support turning off classification or detection in GUI (https://github.com/brainglobe/cellfinder/pull/402)
- Support for single z-stack tif file for input (https://github.com/brainglobe/cellfinder/pull/397)
- Cell detection optimisation (#398) by (https://github.com/brainglobe/cellfinder/pull/407)
- Pretrained models moving to `~.brainglobe/cellfinder`, how the old models can be deleted, and how to re-install them in the new location (#176) (https://github.com/brainglobe/cellfinder/pull/414)
- Maybe cell detection in torch.

cc @alessandrofelder

### PR DESCRIPTION
**Pull request by IgorTatarnikov**
Before submitting a pull request (PR), please read the [contributing guide](https://github.com/brainglobe/.github/blob/main/CONTRIBUTING.md).

Please fill out as much of this template as you can, but if you have any problems or questions, just leave a comment and we will help out :)

## Description

**What is this PR**

- [ ] Bug fix
- [ ] Addition of a new feature
- [x] Other

**Why is this PR needed?**
We're switching from TensorFlow to a PyTorch backend in `cellfinder` v1.3.0. This is a major change and should have a blog post along side the release.

**What does this PR do?**

## References
Closes #177 

## How has this PR been tested?
Build locally and visually inspected.

## Is this a breaking change?
No

## Does this PR require an update to the documentation?
No

If any features have changed, or have been added. Please explain how the documentation has been updated (and link to the associated PR). See [here](https://brainglobe.info/community/developers/index.html#to-improve-the-documentation) for details.

## Checklist:

- [x] The code has been tested locally
- [ ] Tests have been added to cover all new functionality (unit & integration)
- [ ] The documentation has been updated to reflect any changes
- [x] The code has been formatted with [pre-commit](https://pre-commit.com/)

### CODE REVIEW

File: docs/source/blog/cellfinder-v1_3.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+ * The default batch size used for detection has been increased to 64, which improves classification speed by approximately 40% on most systems. The batch size used for detection can be adjusted in the `napari` plugin.
+
+## What do I need to do?
+
+Since this is a major update, we recommend using a fresh conda environment.
+For GPU support, please follow the installation instructions in the [documentation](../documentation/setting-up/gpu.md).
+
+```bash
+conda create -n cellfinder -c conda-forge python=3.10
```
Comment (adamltyson) [inline docs/source/blog/cellfinder-v1_3.md:26]: May as well start the process of moving to 3.11

```suggestion
conda create -n cellfinder -c conda-forge python=3.11
```

File: docs/source/blog/cellfinder-v1_3.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+You can also update an existing installation of `cellfinder` using pip:
+
+```bash
+pip install --upgrade cellfinder
+```
+
+
+## Classification performance
+The classification performance between the two versions is comparable. Below is a comparison of the performance between the two versions using data from the [`cellfinder` paper](https://doi.org/10.1371/journal.pcbi.1009074). Running `cellfinder` with a PyTorch backend results in a  comparable Pearson correlation and slightly improved linear best-fit slope (labelled as "coeff" in the plot) when comparing to manual cell counts. For more details on how the graphs were generated, see the [`cellfinder` paper](https://doi.org/10.1371/journal.pcbi.1009074).
```
Comment (adamltyson) [inline docs/source/blog/cellfinder-v1_3.md:39]:
```suggestion
The classification performance between the two versions is comparable. Below is a comparison of the performance between the two versions using data from the [`cellfinder` paper](https://doi.org/10.1371/journal.pcbi.1009074). Running `cellfinder` with a PyTorch backend results in a  comparable Pearson correlation and slightly improved linear best-fit slope (labelled as "coeff" in the plot) when comparing to manual cell counts. For more details on how the plots were generated, see the [`cellfinder` paper](https://doi.org/10.1371/journal.pcbi.1009074).
```

File: docs/source/blog/cellfinder-v1_3.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+### TensorFlow backend
+![TensorFlow Performance](../_static/comparison_tensorflow.png)
+
+### PyTorch backend
+![PyTorch Performance](../_static/comparison_torch.png)
```
Comment (adamltyson) [inline docs/source/blog/cellfinder-v1_3.md:45]: Can these plots be made a bit smaller? They seem disproportionately large

Comment (IgorTatarnikov) [reply docs/source/blog/cellfinder-v1_3.md]: Done, they're now max 800 px in width. I've centered them for now but I'm not sure if that's better than just left aligning them.

**Pull request merged**